mesonbuild/interpreterbase.py

```diff
@@ -144,7 +144,7 @@ def wrapped(*wrapped_args, **wrapped_kwargs):
     return wrapped
 
 def noArgsFlattening(f):
-    setattr(f, 'no-args-flattening', True)
+    setattr(f, 'no-args-flattening', True) # noqa: B010
```

Comment:
usually python style is to have two spaces between the last code line and the `#` in an inline comment.

Reply:
Sure, but this is a magic comment that disables the `B010` and it has to be on the same line as the function.

Also, as far as I know, python doesn't have inline comments. I am not sure what you want me to change? Do you want me to add another ` ` in front of the `#`?

Reply:
Nevermind, I found https://www.python.org/dev/peps/pep-0008/#inline-comments. I thought you were talking about C style comments.